runtime/cere/src/lib.rs

```diff
@@ -557,7 +559,9 @@ impl pallet_fast_unstake::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type ControlOrigin = frame_system::EnsureRoot<AccountId>;
 	type Deposit = ConstU128<{ DOLLARS }>;
-	type DepositCurrency = Balances;
+	type Currency = Balances;
+	type BatchSize = frame_support::traits::ConstU32<64>;
```

Comment:
If an upgrade introduces a new configuration parameter, it would be nice to briefly explain how it should be configured. In this particular case, why 64 is chosen?

Reply:
Since we're using substrate and trying to be compatible with their changes across different releases, we have to make changes to our code to match those. Which means that i don't get to choose the values but rather port them from substrate / polkadot:
https://github.com/paritytech/polkadot/blob/c7d6c21242fc654f6f069e12c00951484dff334d/runtime/polkadot/src/lib.rs#L624